### PR TITLE
license request page updates

### PR DESCRIPTION
**Pull request by BassCoder2808**
- Intended to solve #452 
- Changed the `submit_new_license` page text once the license is successfully submitted

### CODE REVIEW

**Comment by BassCoder2808** [comment]
Hi @jlovejoy I have updated the PR, please let me know if anything needs to be changed/altered in this

File: src/app/views.py [hunk 1]
```diff
@@ -162,7 +163,9 @@ def submitNewLicense(request):
                         licenseId = licenseRequest.id
                         serverUrl = request.build_absolute_uri('/')
                         licenseRequestUrl = os.path.join(serverUrl, reverse('license-requests')[1:], str(licenseId))
-                        statusCode = utils.createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseComments, licenseSourceUrls, licenseHeader, licenseOsi, licenseExamples, licenseRequestUrl, token, urlType)
+                        statusCode, githubIssueId = utils.createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseComments, licenseSourceUrls, licenseHeader, licenseOsi, licenseExamples, licenseRequestUrl, token, urlType)
```
Comment (rtgdk) [inline src/app/views.py:166]: @BassCoder2808  `utils.createIssue(` is used elsewhere as well, need to update those too.

Comment (BassCoder2808) [reply src/app/views.py:166]: Hi @rtgdk , I have made the changes please let me know, if anything else needs to be changed in this

File: src/app/migrations/0002_licenserequest_issueid.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-03-31 08:23
+
+from django.db import migrations, models
```
[the remark this answers is not in the document]
Comment (BassCoder2808) [reply src/app/migrations/0002_licenserequest_issueid.py:3]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (BassCoder2808) [reply src/app/migrations/0002_licenserequest_issueid.py:3]: @sonatype-lift ignore

File: src/app/utils.py [hunk 3]
```diff
@@ -352,7 +352,7 @@ def createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseCommen
     headers = {'Authorization': 'token ' + token}
     url = "{0}/issues".format(TYPE_TO_URL_LICENSE[urlType])
     r = requests.post(url, data=json.dumps(payload), headers=headers)
-    return r.status_code
+    return r.status_code, r.json()["number"]
```
Comment (rtgdk) [inline src/app/utils.py:355]: Remove common code with #450

File: src/app/templates/app/license_information.html [hunk 4]
```diff
@@ -44,8 +44,10 @@
         <p>{{licenseInformation.notes}}</p>
     </div>
     <div class="licenseField col-sm-12">
-        <p class="fieldName">Standard License Header</p>
-        <p>{{licenseInformation.standardLicenseHeader}}</p>
+        <p class="fieldName">Github Issue</p>
+        <a href="https://github.com/spdx/licnse-list-xml/issues/{{licenseInformation.issueId}}" target="_blank">
```
Comment (rtgdk) [inline src/app/templates/app/license_information.html:48]: Let's make this url prefix a constant.

**Review by rtgdk — CHANGES_REQUESTED** [review]
Overall LGTM, left some comments to address before merge.

File: src/app/views.py [hunk 5]
```diff
@@ -353,6 +360,8 @@ def licenseInformation(request, licenseId):
     licenseInformation['notes'] = data['notes']
     licenseInformation['standardLicenseHeader'] = data['standardLicenseHeader']
     licenseInformation['text'] = data['text']
+    licenseInformation['issueId'] = licenseRequest.issueId
+    licenseInformation['issueUrl'] = "https://github.com/spdx/licnse-list-xml/issues/" + str(licenseRequest.issueId)
```
[the remark this answers is not in the document]
Comment (BassCoder2808) [reply src/app/views.py:364]: @sonatype-lift ignoreall

**Comment by goneall** [comment]
@BassCoder2808 - @rtgdk requested a couple of changes for this PR - if you could update the PR I'll merge this in for the next release.

**Comment by BassCoder2808** [comment]
Hi @goneall I have merged the conflits which were there and also have made the changes which @rtgdk had requested in this PR, let me know if anything else needs to be changed in this

**Comment by goneall** [comment]
Looks like the comments from @rtgdk has been addressed and the merge conflicts have been resolved.

I'll go ahead and merge.

**Comment by goneall** [comment]
This ended up causing issue #483 - Reverting the merge.  @BassCoder2808 - if you could address the issue and test on your local machine to make sure everything is working.

**Pull request merged**